### ISSUE TITLE
blockSelected of too distant block gets overwritten with dayChanged

### ISSUE THREAD

**Issue by tomasvrana** [input-output-hk/symphony#1]
When one selects block from another day then `blockSelected` gets overwritten with `dayChanged` before the "opening" animation finishes. 

Results in no data shown for that block.

**Comment by scottdarby** [comment]
this should be resolved now, let me know if you still see it happening

**Comment by tomasvrana** [comment]
Yeah I do @scottdarby ... still on my end. When selecting block behind the date divider. It loads the block data however when passing thru the divider it gets replaced with day data. Don't see any change. Just get the `dayChanged` ignored if there still block sliding in.
